ansible/roles/debops-contrib.etckeeper/tasks/main.yml

```diff
@@ -111,6 +111,14 @@
   tags: [ 'role::etckeeper:vcs_config' ]
 ## ]]]
 
+- name: Unstage ignored files from Git index
+  shell: etckeeper vcs ls-files -i --exclude-standard -z | xargs -0 etckeeper vcs reset --mixed --
+  register: git_reset
+  when:
+    - etckeeper__vcs == "git"
+    - etckeeper__register_init|changed
```

Comment:
Hmm, I wasn't aware that syntax could work... I need to check it out.

Reply:
I think the filter `changed` just checks `etckeeper__register_init.changed` property. I am new to Ansible. I took it from task "Create root commit" (right after this one):

```yaml
- name: Create root commit
  command: etckeeper commit 'Initial commit created by Ansible.'
  when: etckeeper__register_init|changed
```

Reply:
No, I meant use of YAML list syntax in a `when:` statement. Apparently it works. Maybe I'll start using it more then.